reference/storefront_tokens.v3.yml

```diff
@@ -179,7 +184,6 @@ components:
           example: 1885635176
           minimum: 0
       required:
-        - channel_id
```

Comment:
Right now it is a little bit tricky, as we expect to receive either channel_id or channel_ids. Not sure how to write it

Reply:
I updated the schema to accommodate this option.